src/orders/components/OrderFulfillPage/OrderFulfillPage.tsx

```diff
@@ -430,7 +430,7 @@ const OrderFulfillPage: React.FC<OrderFulfillPageProps> = props => {
                                     !!errors?.find(
                                       err =>
                                         err.warehouse === warehouse.id &&
-                                        err.orderLine === line.id &&
+                                        err.orderLines[0] === line.id &&
```

Comment:
I'm pretty sure you should search in whole array, not just 0th index

Reply:
Jus to chip in - this is a very temporary workaround to make backend changes mergeable and further frontend implementation will be worked on immediately after this, but I agree this is a very low-effort change so let's change it now